scripts/image_loader.js

```diff
@@ -181,7 +181,7 @@ settingsLoadedEvent.addHandler(function()
             {
                 var video_id;
                 
-                if ((video_id = href.match(/i.imgur\.com\/(\w+)/i)))
+                if ((video_id = href.match(/[i.]?imgur\.com\/(\w+)/i)))
```

Comment:
I did test it, and yes it does not appear to behave like .test() does up above. Regardless, I think I am just going to remove it as its superfluous anyway.